### PR TITLE
[r] Update requirements

### PR DESCRIPTION
**Pull request by catriuspham**
### Description

The title says it all

### Notes

This does not change what is installed during "pip install". Just for deverloping purposes.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
-mock==2.0.0
-termcolor==1.1.0
+coverage
+flake8
+mock
+nose
+termcolor
+tox
```
Comment (steadykhoi) [inline requirements.txt]: Why is there no package version? Like this we open ourselves to the risk of bug due to package version changes.

Comment (catriuspham) [reply requirements.txt]: I thought about that too. However, we have been installing these packages manually without the requirements file from the very first day. And up to now, we're still good.

Comment (steadykhoi) [reply requirements.txt]: Yes I believe that. Still doesn't mean we're future proof. So if you find the current versions are working for you, you should lock that down now.

**Review by hieueastagile — APPROVED** [review]
LGTM

**Comment by catriuspham** [comment]
@oyster Please help merge this PR thanks.

**Pull request merged**